.github/workflows/codespell.yml

```diff
@@ -0,0 +1,23 @@
+# Codespell configuration is within .codespellrc
+---
+name: Codespell
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
```

Comment:
Not familiar with codespell, but shouldnt this be run as part of PRs that target develop? Then typos can be caught early.

Reply:
that is what it says above -- run on pull requests that target develop (as well as pushes to develop) ;)